packages/terra-core-docs/src/terra-dev-site/doc/badge/example/BadgeIntent.jsx

```diff
@@ -6,29 +6,71 @@ import styles from './Badge.module.scss';
 const cx = classNames.bind(styles);
 
 const BadgeIntent = () => (
-  <div className={cx('badge-container')}>
-    <div className={cx('badge')}>
-      <Badge text="Patient Arrival" />
+  <React.Fragment>
+    <p>
+      Intent with severity of issues.
+    </p>
+    <div className={cx('badge-container')}>
+      <div className={cx('badge')}>
+        <Badge text="No Risk" />
+      </div>
+      <div className={cx('badge')}>
+        <Badge intent="primary" text="Low" />
+      </div>
+      <div className={cx('badge')}>
+        <Badge intent="secondary" text="Moderate" />
+      </div>
+      <div className={cx('badge')}>
+        <Badge intent="warning" text="High" />
+      </div>
+      <div className={cx('badge')}>
+        <Badge intent="negative" text="Critical" />
+      </div>
     </div>
-    <div className={cx('badge')}>
-      <Badge intent="primary" text="Admitted" />
+    <p>
+      Secondary Intent can be used to label user name as NEW
+    </p>
+    <div className={cx('badge-container')}>
+      <div className={cx('badge')}>
+        <React.Fragment>Segun Adebayo </React.Fragment>
+        <Badge intent="secondary" text="NEW" />
```

Comment:
User Name example is not clear !! we need to have banner or visible container around it. (instead of user pic we can make use of terra-icon OR terra-avatar for user icon.) 
when we screen reader focus on that it should provide information about the badge that has been labelled as new. ( we can use visuallyHiddenText of terra-badge to provide the additional context of terra-badge. In this scenario If screen readers read the user example as 'UserName, New User' it will sound better for SR user. ) Or you can choose better suitable information.

<img width="789" alt="Screenshot 2023-10-31 at 12 28 52 PM" src="https://github.com/cerner/terra-core/assets/12869809/18526e19-4fa0-444a-9feb-0e956e737672">

Reply:
changes done Thanks